src/views/webview-app/components/connect-form/authentication/kerberos.tsx

```diff
@@ -150,15 +150,14 @@ class Kerberos extends React.Component<props> {
               <span>Canonicalize Host Name</span>
             </label>
           </div>
-          <Toggle
+          {/* <Toggle
             className={styles['form-toggle']}
             name="kerberosCanonicalizeHostname"
             onChange={this.onCnameToggle}
             checked={kerberosCanonicalizeHostname || false}
             size="small"
-            variant="default"
             disabled={false}
-          />
+          /> */}
```

Comment:
Is this temporary? I’m obviously not having the full context here, but it’s not obvious from the diff why this is being commented out :)

Reply:
ah ty meant to mention that before opening the pr hah

Reply:
Added it back in, moved PR out of WIP. Had to add the `esModuleInterop` flag in tsconfig to get the leafygreen import working right, cleaned up some of our other imports for that to work.